get-started/create-run.qmd

```diff
@@ -0,0 +1,77 @@
+---
+title: "Create and Run"
+---
```

Comment:
I'm not sure if this is 100% correct, but the intention is to redirect this link

https://shiny.posit.co/py/docs/install-create-run.html

```suggestion
---
title: "Create and Run"
aliases:
  - docs/install-create-run.html
---
```

Reply:
Oh nevermind, I see now that this links to `install.qmd`, which makes sense